src/js/htmlutils.js

```diff
@@ -115,11 +115,11 @@ let htmlUtils = {
       return `
 <div class="switch-container ${action}">
   <div class="switch-toggle switch-3 switch-candy">
-    <input id="block-${origin_id}" name="${origin}" value="${constants.BLOCK}" type="radio" ${is_checked(constants.BLOCK, action)}>
+    <input id="block-${origin_id}" name="${origin}" value="${constants.BLOCK}" type="radio" aria-label="${i18n.getMessage("domain_slider_block_tooltip")}: ${origin}" ${is_checked(constants.BLOCK, action)}>
     <label title="${tooltips.block}" class="tooltip" for="block-${origin_id}"></label>
-    <input id="cookieblock-${origin_id}" name="${origin}" value="${constants.COOKIEBLOCK}" type="radio" ${is_checked(constants.COOKIEBLOCK, action)}>
+    <input id="cookieblock-${origin_id}" name="${origin}" value="${constants.COOKIEBLOCK}" type="radio" aria-label="${i18n.getMessage("domain_slider_cookieblock_tooltip")}: ${origin}" ${is_checked(constants.COOKIEBLOCK, action)}>
```

Comment:
Concatenating localized strings by hand doesn't work for right-to-left locales (or locales where grammar prescribes a different sentence order).

Do we need to have the domain be part of the ARIA label for the input? Or could we have another (ARIA?) attribute that communicates the domain the ARIA label refers to? Just thinking out loud.

Reply:
Unfortunately as it is now (at least with the VoiceOver feature on MacOS I'm currently testing on) the domain for each tracker isn't "next" to the sliders, so that in a longer list of trackers, it's very difficult to know which domain the current slider row belongs to. 

I wonder if it isn't too much a logical misstep for right-to-left locales to simply have it be `<domain> : <slider state description>` and for left-to-right be `<slider state description> : <domain>` since the order of each text element there doesn't matter as much as the colon denoting their relationship to each other?

Reply:
I got the sense from https://github.com/EFForg/privacybadger/issues/2764#issuecomment-854239571 that it's clear what the relevant domain is just by virtue of our DOM structure. Which means no need to work the domain into the ARIA label.